Source/Core/Core/Scripting/HelperClasses/ArgHolder.h

```diff
@@ -0,0 +1,87 @@
+#ifndef ARG_HOLDER
+#define ARG_HOLDER
+
+#include <imgui.h>
+#include <string>
+#include <vector>
+#include "Common/CommonTypes.h"
+#include "Core/Movie.h"
+#include "Core/Scripting/CoreScriptInterface/Enums/ArgTypeEnum.h"
+
+namespace Scripting
+{
+
+// This is the implementation for the ArgHolder struct which is referenced by ArgHolder_APIs
+struct ArgHolder
+{
+  ArgTypeEnum argument_type;
+  bool contains_value;  // True if the ArgHolder contains a value, and false if it's empty.
```

Comment:
If you put this before the member (rather than at the end of the line), it also shows up in various IDEs as documentation.
```suggestion
  // True if the ArgHolder contains a value, and false if it's empty.
  bool contains_value;
```